lib/ansible/modules/cloud/amazon/cloudformation.py

```diff
@@ -596,6 +598,16 @@ def main():
 
     result = {}
 
+    # validate tags
+    if module.params.get('tags'):
+        team_tag = module.params['tags']['Team']
+        project_tag = module.params['tags']['Project']
+        validation_res = tag_validator.validate(team_tag, project_tag)
```

Comment:
Minor comment, `Name` tag is also required, but free form
https://dataxu.atlassian.net/wiki/spaces/dev/pages/540606620/AWS+Tags+2.0

Reply:
Thanks @jmcorallo I was aware of the Name tag. 
@babineaum @scobrc do we also need to validate Name tag, can you pls clarify?

Reply:
Name tag will not be validated.